gym_quadruped/quadruped_env.py

```diff
@@ -20,6 +20,8 @@
 from gym_quadruped.utils.quadruped_utils import (LegsAttr, configure_observation_space_representations,
                                                  configure_observation_space, extract_mj_joint_info)
 
+from gym_quadruped.utils.mujoco.terrain import add_world_of_boxes, add_world_of_pyramid
```

Comment:
These imports should be local only if the chosen terrains are requested. This because each random terrain generator might possess dependencies that are unnceseary if this type of terrain is not needeed.